src/node_worker.h

```diff
@@ -90,6 +98,9 @@ class Worker : public AsyncWrap {
   // instance refers to it via its [kPort] property.
   MessagePort* parent_port_ = nullptr;
 
+  LoopStopper* trigger_ = nullptr;
+  LoopStopper* response_ = nullptr;
```

Comment:
- Can we give these names that are more expressive about what they do, e.g. `thread_stopper_` and `on_thread_finished_`?
- We need to free these objects, otherwise this is currently a memory leak.

Reply:
done, renamed.